neard/src/cli.rs

```diff
@@ -239,6 +244,8 @@ pub(super) enum NeardSubCommand {
 
     /// validate config files including genesis.json and config.json
     ValidateConfig(ValidateConfigCommand),
+
```

Comment:
Please add a comment, because this comment will be displayed when running `./neard` without a sub-command.